sound/hda/intel-sdw-acpi.c

```diff
@@ -57,19 +57,18 @@ static int
 sdw_intel_scan_controller(struct sdw_intel_acpi_info *info)
 {
 	struct acpi_device *adev = acpi_fetch_acpi_dev(info->handle);
+	unsigned long list_ul;
+	u32 list;
```

Comment:
nitpick: And here, `u32 list;` after `u32 count;`

Reply:
done